packages/@okta/vuepress-site/guides/build-self-signed-jwt/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Build a Self-Signed JWT
```

Comment:
How aboutBuild a JWT for Client Authentication? In the future there may be other unrelated reasons to build a signed JWT.